tests/Unit/GuzzleFileFetcherTest.php

```diff
@@ -35,9 +35,17 @@ class GuzzleFileFetcherTest extends TestCase
     /**
      * The JSON content of the mocked response(s).
      *
+     * Deliberately includes whitespace in order to prove that the fetcher does
+     * not alter the JSON in any way.
+     *
      * @var string
      */
-    private $testContent = '["oolong","assam","matcha","herbal"]';
+    private $testContent = <<<END
+["oolong",
+  "assam",
+"matcha",
+"herbal"]
+END;
```

Comment:
Could you make the test data an object with keys and the keys be unsorted also? Or have 2 test cases 1 with unsorted keys and 1 with whitespace. 

The previous bug would have showed up in the unsorted keys case also even if there was no whitespace

Reply:
Done.